openff/toolkit/tests/test_forcefield.py

```diff
@@ -4322,6 +4321,72 @@ def test_overwrite_bond_orders(self):
         ):
             assert bond1.bond.fractional_bond_order == bond2.bond.fractional_bond_order
 
+    def test_fractional_bond_order_ignore_existing_confs(self):
+        """Test that previously-defined bond orders in the topology are overwritten"""
+        mol = create_ethanol()
+        top = Topology.from_molecules(mol)
+
+        mod_mol = create_ethanol()
+        mod_mol.generate_conformers()
+        mod_mol._conformers[0][0][0] = (
+            mod_mol._conformers[0][0][0] + 1.0 * unit.angstrom
+        )
+        mod_mol._conformers[0][1][0] = (
+            mod_mol._conformers[0][1][0] - 1.0 * unit.angstrom
+        )
+        mod_mol._conformers[0][2][0] = (
+            mod_mol._conformers[0][2][0] + 1.0 * unit.angstrom
+        )
```

Comment:
You might consider compacting this and avoiding the private access:

```suggestion
        mod_mol.conformers[0] += np.random.random((mod_mol.n_atoms, 3)) * unit.angstrom
```

Reply:
I like how concise this is, but I'm a little concerned that using random values will occasionally cause weird behavior, so I'm going to keep the geometry manipulations explicit here.